README.md

```diff
@@ -79,38 +79,36 @@ sudo ./install-dependencies.sh && CC=clang CXX=clang++ ./build.sh
 
 For quicker dev setup, we provide a [docker image](tools/docker/README.md) with the toolchain installed.
 
-# Beta builds
+# Release candidate builds
```

Comment:
Not sure everyone knows what a release candidate build is. How about we call these "Testing" or "Non production" builds?

Reply:
This is the readme on GitHub not public docs. RC makes sense to me. Although we can say that in the description

Reply:
Pretty sure "release candidate" is a common enough name for the non-supported builds which contain expected functionality prior to the final release. Agreed we should describe it clearly in the description (along with the difference between beta builds which are not necessarily all the expected functionality as the final version).

Reply:
We might want to add a sentence like "RC builds are not recommended for production use"